src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -438,6 +438,20 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
             downQueries.push(new Query(`ALTER TABLE ${this.escapePath(newTable)} RENAME CONSTRAINT "${newPkName}" TO "${oldPkName}"`));
         }
 
+        // rename sequences
+        newTable.columns.map(col => {
+            if (col.isGenerated) {
```

Comment:
you need `&& col.generationStrategy !== "uuid"` here, because uuid columns does not have sequences

Reply:
Thank you, @AlexMesser 
If that code is supposed to be only effective to "increment" strategy, wouldn't it be better `col.generationStrategy === "increment"`? We can avoid an extra condition of `undefined`.

Reply:
yes, for Postgres we can only check for `increment`